### PR TITLE
Add naughty for libvirt crash in debian stable

### PR DESCRIPTION
**Pull request by KKoukiou**
See known issue #1372

### CODE REVIEW

**Comment by KKoukiou** [comment]
> See known issue #1372

Related failure https://logs.cockpit-project.org/logs/pull-1350-20201103-090725-fe2ff21e-debian-stable-cockpit-project-cockpit/
This is from where I fetched the core files also for the bug report.

**Comment by KKoukiou** [comment]
@marusak I forgot how to locally test test output agains naughties to check that the regexp applies. Do you remember the command to run for that?

**Comment by martinpitt** [comment]
@KKoukiou: copy the raw test output into a file, then go into bots, and run `cat /tmp/output | ./tests-policy -o centos-8-stream` (or whichever image). It should turn a FAIL into a SKIP if a pattern matches.

File: naughty/debian-stable/1372-libvirt-crashes-on-test-teardown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*testAddDiskNFS*
+*
+Process * (libvirtd) of user * dumped core.
+*
+#0  * __GI___pthread_mutex_lock (libpthread.so.0)
+#1  * virQEMUDriverGetConfig (libvirt_driver_qemu.so)
+#2  * n/a (libvirt_driver_qemu.so)
```
Comment (martinpitt) [inline naughty/debian-stable/1372-libvirt-crashes-on-test-teardown:7]: I suggest to skip lines #0 and #2, as there might be too much fuzz. Just matching on #1 and #3 should be enough.

Comment (KKoukiou) [reply naughty/debian-stable/1372-libvirt-crashes-on-test-teardown:7]: It was actually skipped as expected - see https://logs.cockpit-project.org/logs/pull-1373-20201104-165549-f6b48ae6-debian-stable-cockpit-project-cockpit/log.html

Shall I still change it or keep it as specific as it get?

Comment (KKoukiou) [reply naughty/debian-stable/1372-libvirt-crashes-on-test-teardown:7]: HA testInlineConsoleWithUrlRoot fails with the exact same way - I think i will need to remove the test pattern match.

**Review by martinpitt — APPROVED** [review]
Bots were down last night, Sanne fixed them. So re-triggering tests. Looks great now, thanks!

**Pull request merged**